mobile-test-app/ios/LLToolTests/LLToolTests.m

```diff
@@ -11,7 +11,7 @@
 #import <React/RCTLog.h>
 #import <React/RCTRootView.h>
 
-#define TIMEOUT_SECONDS 3600
+#define TIMEOUT_SECONDS 900
```

Comment:
maybe not enough

Reply:
yeah don't worry i was testing with just a few tests